### PR TITLE
Brnadon createpresenter6

### PR DESCRIPTION
**Pull request by branflake2267**
Working on create presenter.
1. Formatting took place, seemed I had 2 spaces. 
2. In the process of adding select content slot dialog.

Up next, I need to drill the scope of the dialog. I set it to all classes to get things tested.

screen shot 2013-09-26 at 5 35 56 pm

### CODE REVIEW

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java [hunk 1]
```diff
@@ -1,243 +1,271 @@
...
+
+    private void initTabFoldersHandlers() {
+        tabbedPanel.addChangeListener(new ChangeListener() {
+            @Override
+            public void stateChanged(ChangeEvent e) {
+                int selectedIndex = tabbedPanel.getSelectedIndex();
 
+                setPresenterType(selectedIndex);
+
+                if (selectedIndex == 0) {
```
Comment (branflake2267) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:248]: I suppose I could drill this some more :)

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java [hunk 2]
```diff
@@ -1,243 +1,271 @@
 package com.arcbees.plugin.idea.wizards.createpresenter;
 
+import com.arcbees.plugin.idea.domain.PresenterConfigModel;
```
Comment (spg) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:4]: Maybe you should rearrange the import layout to : 

```
import java.*
<blank line>
import javax.*
<blank line>
import org.*
<blank line>
import all other imports
<blank line>
import static all other imports
```

This is the "standard" import layout we use on many projects

Comment (spg) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:4]: And regarding you star import problem, maybe you should remove all entries from `Preferences > Code Style > Java > Imports > Packages to Use Import with '*'`

**Comment by spg** [comment]
LGTM

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java (hunk 2 above)
Comment (branflake2267) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:4]: Thanks for that info. :)

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java [hunk 3]
```diff
@@ -1,243 +1,271 @@
...
+    private JCheckBox useSingleton2;
+    private JCheckBox useOverrideDefaultPopup;
+    private JCheckBox useAddOnhide;
+    private JCheckBox useAddOnreset;
+    private JCheckBox useAddOnunbind;
+    private JLabel lblQuerystring;
+
+    public CreatePresenterForm(Project project) {
+        super(project);
+        init();
```
Comment (meriouma) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:67]: nl after super

File: src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java [hunk 4]
```diff
@@ -1,243 +1,271 @@
...
+                }
+            }
+        });
+    }
+
+    private void initRadioHandlers() {
+        radioNestedPresenter.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                boolean selected = radioNestedPresenter.isSelected();
```
Comment (meriouma) [inline src/com/arcbees/plugin/idea/wizards/createpresenter/CreatePresenterForm.java:207]: Create a method

```
private void onPresenterRadioWidgetSelected(JRadioButton radioButton, int index) {
  boolean selected = radioButton.isSelected();
  if (selected) {
    setSelectedIndex(index);
  }
}
```

Then you can replace those 3 `actionPerformed()` with : `onPresenterRadioWidgetSelected(radioNestedPresenter, 0);`

**Comment by meriouma** [comment]
Did you change code style from 2 to 4 spaces everywhere?

**Comment by branflake2267** [comment]
It should be four. My bad on to. Was happening during project changes. Now
fixed, because i copied style.

**Pull request merged**